packages/playground/wordpress/src/index.ts

```diff
@@ -131,27 +146,29 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 			if (!$user) {
 				return;
 			}
+			/**
+			 * This approach is described in a comment on
+			 * https://developer.wordpress.org/reference/functions/wp_set_current_user/
+			 */
 			wp_set_current_user( $user->ID, $user->user_login );
 			wp_set_auth_cookie( $user->ID );
 			do_action( 'wp_login', $user->user_login, $user );
 			setcookie('playground_auto_login_already_happened', '1');
-		}
 
+			/**
+			 * Reload page to ensure the user is logged in correctly.
+			 * WordPress uses cookies to determine if the user is logged in,
+			 * so we need to reload the page to ensure the cookies are set.
+			 */
+			wp_redirect($_SERVER['REQUEST_URI']);
+			exit;
```

Comment:
Adding this `exit` after redirecting break Safari. Playground doesn't load with the error `Failed to load resource: The URL can’t be shown`.

If I remove the exit other browsers start having issues.
I'm taking a look at why is Safari failing and other browsers aren't.

Reply:
It seems to be related to how different browsers handle 302 responses. Chrome renders the response, while Safari doesn't even show the request the list of network requests.

When I force the response status to be 200 using `http_response_code` Safari starts working, but Chrome stops.

Reply:
I made a minimal test using PHP and iframes. The reloaded page was able to load in a Safari `iframe`. 

I assume the problem is in the response returned by Playground.
I will try to compare these responses and see what's causing the issue.

Reply:
It's in the redirect code. The new redirect is running multiple times and this is causing issues.

Reply:
What about adding something like this at the top of the function:
```php
$should_redirect = empty( $_COOKIE ) && ! empty($_SERVER['REQUEST_URI']);
```

Reply:
The problem was in the `$_SERVER['REQUEST_URI']`, it's relative. When I switch it to an absolute URL it works.